docs/documentation/maya/skin.md

```diff
@@ -1,127 +1,189 @@
-# How To Use
+# AdnSkin
 
-Skin is a Maya deformer for fast, robust and easy-to-configure skin simulation for digital assets. Thanks to the combination of internal and external constraints, the deformer can produce dynamics that allow the skin mesh to realistically react to the deformations of the internal tissues (e.g. muscles, fascia) over time.
+AdnSkin is a Maya deformer for fast, robust and easy-to-configure skin simulation for digital assets. Thanks to the combination of internal and external constraints, the deformer can produce dynamics that allow the skin mesh to realistically react to the deformations of the internal tissues (e.g. muscles, fascia) over time.
+
+<!-- DEV NOTE: change paint tool references once tools.md has been merged-->
+
+The influence these constraints have on the simulated mesh can be freely modified by painting them via the [AdonisFX Paint Tool](index.md) or by uniformingly regulating their influence via multipliers in the attribute editor. Besides the maps and multipliers there are many other parameters to regulate the skin's dynamics and behaviour to a wide array of options.
```

Comment:
In my opinion this note should be handled in this PR. The merge update with `wip_v1.0.0` should have included the `tools.md` file.

I suggest to:
1. Update this branch with `wip_v1.0.0`.
2. Handle all references that might be missing.
3. Remove DEV NOTE found.
```suggestion
The influence these constraints have on the simulated mesh can be freely modified by painting them via the [AdonisFX Paint Tool](tools.md#adonisfx-paint-tool) or by uniformingly regulating their influence via multipliers in the attribute editor. Besides the maps and multipliers there are many other parameters to regulate the skin's dynamics and behaviour to a wide array of options.
```

Reply:
Merged, references fixed and DEV NOTE removed!